packages/@aws-cdk/aws-lambda/lib/log-retention.ts

```diff
@@ -26,6 +26,31 @@ export interface LogRetentionProps {
    * @default - A new role is created
    */
   readonly role?: iam.IRole;
+
+  /**
+   * Retry options for managing CloudWatch log groups.
+   *
+   * @default - AWS SDK default retry options
+   */
+  readonly logRetentionRetryOptions?: LogRetentionRetryOptions;
+}
+
+/**
+ * Retry options for managing CloudWatch log groups
+ */
+export interface LogRetentionRetryOptions {
+  /**
+   * The maximum amount of retries.
+   *
+   * @default - AWS SDK default
```

Comment:
We usually document the default, even if it comes from a dependency

Reply:
Hmmm, yeah I can change that, but left it like this on purpose. How will you make sure it stays in sync?

Reply:
It's unlikely the SDK's defaults change. Changing so may break existing customers using the CLI whose applications work because of a given set of defaults, so this is likely to be considered a breaking change.

It may change on the next major version but we have better control over that from the import statements.